src/main/java/frc/robot/subsystems/Subsystems.java

```diff
@@ -29,8 +38,11 @@ public class Subsystems {
           Constants.APRILTAG_ROBOT_TO_CAMERA1);
 
   ArrayList<Subsystem> all =
-      new ArrayList<Subsystem>(Arrays.asList(drivetrain, elevator, coralArm));
-  ArrayList<Subsystem> manipulators = new ArrayList<Subsystem>(Arrays.asList(elevator, coralArm));
+      new ArrayList<Subsystem>(
+          Arrays.asList(drivetrain, elevator, coralArm, algaeArm, algaeGrabber, coralRoller));
```

Comment:
#44 changes how the `all` and `manipulators` lists are managed. Please rebase and update.